src/katsdpcontroller/defaults.py

```diff
@@ -51,6 +51,9 @@
 #: Default payload size for gpucbf data products. Bigger is better to
 #: minimise the number of packets/second to process.
 GPUCBF_PACKET_PAYLOAD_BYTES = 8192
+#: Minimum update period (in seconds) for katcp sensors where the underlying
+#: value may update extremely rapidly.
+GPUCBF_MIN_SENSOR_UPDATE_PERIOD = 1.0
```

Comment:
I think let's call it something like `FAST_SENSOR_UPDATE_PERIOD`. We're applying it because it updates too quickly otherwise, not because it's specific to gpucbf.